classes/submission/Collector.php

```diff
@@ -388,8 +390,12 @@ public function getQueryBuilder(): Builder
             $q->whereIn('s.submission_id', array_map(intval(...), $this->submissionIds));
         }
 
+        // Add support to serch using DOI identifiers
+        // search phrases starting number followed by a '.'  will be interpreted as a DOI identifier. E.g: 10.1
+        $pattern = '/^\d+\./i';
```

Comment:
Up to you, but it may be useful to extract this to something like a static method on the DOI class so if we do end up needing to update this regex check in the future, there are fewer places took. Something like `Doi::isValid(string $doiToCheck): bool`...